packages/hooks/src/useCurrentUser.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
```

Comment:
Please add to README

Reply:
I added an example to the README as well as a story for Storybook